src/cli/Cli.ts

```diff
@@ -203,6 +203,23 @@ export class Cli {
       optionsWithoutShorts.options.output = this.getSettingWithDefaultValue<string | undefined>(settingsNames.output, undefined);
     }
 
+    // Validate the Option sets provided in the command
+    const optionsSets: string[][] | undefined = this.commandToExecute.command.optionSets();
+    const argsOptions: string[] = Object.keys(this.optionsFromArgs.options);
+    const cmdArgs = this.optionsFromArgs;
+    optionsSets?.forEach((optionSet) => {
+      if (optionSet.length > 0) {
+        const commonOptions = argsOptions.filter(opt => optionSet.includes(opt));
+        if (commonOptions.length === 0) {
+          return this.closeWithError(`Specify either ${optionSet.map(opt => opt).join(',')}.`, cmdArgs, false);
```

Comment:
Why are we explicitly not showing the help here? Instead we should use the setting that user has defined